radiography/src/main/java/radiography/ScannableView.kt

```diff
@@ -68,9 +70,25 @@ public sealed class ScannableView {
     public val semanticsConfigurations: List<SemanticsConfiguration>
       get() {
         return semanticsNodes.map { it.config }.ifEmpty { null }
+          // Before Compose 1.6, semantics could be held in a SemanticsModifier
           ?: modifiers
             .filterIsInstance<SemanticsModifier>()
             .map { it.semanticsConfiguration }
+            .ifEmpty { null }
+            // After Compose 1.6, semantics are held in nodes. While this usually gets represented
+            // by semanticsNodes, sometimes they instead are only found in SemanticsModifierNode,
+            // which we must then extract.
+          ?: modifiers
+            .filterIsInstance<ModifierNodeElement<*>>()
+            .map { it.create() }
+            .filterIsInstance<SemanticsModifierNode>()
+            .map { semanticsModifierNode ->
+              semanticsModifierNode.run {
+                val semanticsConfiguration = SemanticsConfiguration()
```

Comment:
this creates a new `SemanticsConfiguration()` for each node - I wasn't sure if that was best, or if we should use the same SemanticsConfiguration() instance and apply each semantics node to it. having a list of `SemanticsConfiguration` seems to be consistent with previous behavior though

Reply:
I'm not sure exactly how we'd want to handle semantics merging here. I think the existing Radiography implementation uses merged config, but since it also gets the configuration per-node, it might not be what "merged config" usually means? It might not matter much - since we _are_ already grouping by node, we can probably just always merge semantics for all modifier nodes on the current node. So we'd probably want to share a `SemanticsConfiguration` object for everything on the same node, and make sure we're calling the modifiers in the right order. We might also need to watch out for `clearAndSetSemantics`, but again I'm not sure we care for Radiography's use case.